### PR TITLE
Create new element: checkbox

### PR DESCRIPTION
**Pull request by zdlm**
@esmailpour-hosein @aleks-elkin 
#171

### CODE REVIEW

File: packages/components/checkbox/src/checkbox.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+		left: 0;
+		height: var(--ts-unit);
+		width: var(--ts-unit);
+		border: var(--ts-border);
+		&:after {
+			content: '';
+			position: absolute;
+			display: none;
+			left: var(--ts-unit-quarter);
+			top: 1px;
```
Comment (esmailpour-hosein) [inline packages/components/checkbox/src/checkbox.css]: Is there any other solution for this? If not can you provide some comment about the reason

File: packages/components/checkbox/src/checkbox.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+			width: var(--ts-unit-quarter);
+			height: var(--ts-unit-half);
+			border: solid var(--ts-color-white);
+			border-width: 0 3px 3px 0;
```
Comment (esmailpour-hosein) [inline packages/components/checkbox/src/checkbox.css]: Can these be variable like checkbox-width?

File: packages/components/checkbox/src/checkbox.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+	& .checkbox-container input:checked ~ span {
+		background-color: var(--ts-disabled-background-color);
+		border: var(--ts-disabled-border);
+	}
+}
+
+/* rtl ............................................................*/
+:host([dir='rtl']) {
+	& .checkbox-container {
+		padding-right: var(--ts-unit-plus);
```
Comment (esmailpour-hosein) [inline packages/components/checkbox/src/checkbox.css]: I think you need to reset the `ltr` related properties as well. like padding-left: initial or 0

File: packages/components/checkbox/src/checkbox.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+		}
+
+		render() {
+			return html`
+				<label class="checkbox-container">
+					${this.label}
+					<input
+						type="checkbox"
+						name="${this.name}"
+						value="${this.value}"
```
Comment (esmailpour-hosein) [inline packages/components/checkbox/src/checkbox.js]: I think you need to use `.` before `value` and `name`
https://lit-html.polymer-project.org/guide/template-reference#binding-types

File: packages/components/checkbox/stories/checkbox.stories.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+
+import '@tradeshift/elements.checkbox';
+
+storiesOf('ts-checkbox', module)
+	.addDecorator(withKnobs)
+	.add('default', () => {
+		const label = text('label', 'Label');
+		const disabled = boolean('Disabled', false);
+
```
Comment (esmailpour-hosein) [inline packages/components/checkbox/stories/checkbox.stories.js]: Do we need the value here ? 🤔 or maybe some actions to show the changed value?

**Comment by esmailpour-hosein** [comment]
Also please add the happo tests.

File: packages/components/checkbox/README.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+    </a>
+    <a href="https://npmcharts.com/compare/@tradeshift/elements.checkbox?minimal=true">
+		  <img alt="Downloads per month" src="https://badgen.net/npm/dm/@tradeshift/elements.checkbox" height="20"/>
+		</a>
+		<a href="https://www.npmjs.com/browse/depended/@tradeshift/elements.checkbox">
+		  <img alt="Dependent packages" src="https://badgen.net/npm/dependents/@tradeshift/elements.checkbox" height="20"/>
+		</a>
+</p>
+
+## ➤ How to use it
```
[the remark this answers is not in the document]
Comment (zdlm) [reply packages/components/checkbox/README.md]: 👍

File: packages/components/checkbox/src/checkbox.css (hunk 1 above)
Comment (zdlm) [reply packages/components/checkbox/src/checkbox.css]: I have to make the check mark looks nice.

File: packages/components/checkbox/src/checkbox.css (hunk 2 above)
Comment (zdlm) [reply packages/components/checkbox/src/checkbox.css]: sure

File: packages/components/checkbox/src/checkbox.css (hunk 3 above)
Comment (zdlm) [reply packages/components/checkbox/src/checkbox.css]: 👍

File: packages/components/checkbox/src/checkbox.js (hunk 4 above)
Comment (zdlm) [reply packages/components/checkbox/src/checkbox.js]: 👍

File: packages/components/checkbox/stories/checkbox.stories.js (hunk 5 above)
Comment (zdlm) [reply packages/components/checkbox/stories/checkbox.stories.js]: I don't think so. Most of the time developers don't need to change the value.

File: packages/components/checkbox/src/checkbox.css (hunk 3 above)
Comment (esmailpour-hosein) [reply packages/components/checkbox/src/checkbox.css]: I mean you need to overwrite that in the `rtl` mode

File: packages/components/checkbox/src/checkbox.css (hunk 2 above)
Comment (esmailpour-hosein) [reply packages/components/checkbox/src/checkbox.css]: Okay

**Pull request merged**